bin/zopen-generate

```diff
@@ -110,7 +110,7 @@ mkdir -p ${name}port/patches
 
 # Create external LICENSE
 touch "${name}port/patches/LICENSE" && chtag -tc 819 "${name}port/patches/LICENSE"
-echo "${licenseName}\nView license contents at ${licenseUrl}" > "${name}port/patches/LICENSE"
+printf "${licenseName}\nView license contents at ${licenseUrl}\n" > "${name}port/patches/LICENSE"
```

Comment:
```suggestion
printf "%s\n" "${licenseName}\nView license contents at ${licenseUrl}" > "${name}port/patches/LICENSE"
```
In case there are substitution characters in the vars